pset-iterators.js

```diff
@@ -184,6 +214,22 @@ console.log('Problem 7:')
 
 // Add your code below this line
 
+const shortTermPresidents = presidents.filter((president) => {
+  if (president.left_office !== null) {
+  let startYear = parseInt(president.took_office.split('-')[0])
+  let endYear = parseInt(president.left_office.split('-')[0])
+  let startMonth = parseInt(president.took_office.split('-')[0])
+  let endMonth = parseInt(president.left_office.split('-')[0])
```

Comment:
This definitely work! But looks like your startMonth and endMonth are a little bit redundant here; you can get away without it on this example. [Here](https://github.com/jsd20200127/hw-03-iterators-solution/blob/c31d6d65d8f18a945cb6e59470aca10687d61aa3/pset-iterators.js#L208-L213) is another potential solution for future benefit.